src/Components/Web/src/Microsoft.AspNetCore.Components.Web.csproj

```diff
@@ -6,6 +6,7 @@
     <Description>Support for rendering ASP.NET Core components for browsers.</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IsShippingPackage>true</IsShippingPackage>
+    <RootNamespace>Microsoft.AspNetCore.Components</RootNamespace>
```

Comment:
For the components that were moved without changes, e.g., EditForm, is your opinion that the namespace is already correct? Is there any reason to consider having a Microsoft.AspNetCore.Components.Web namespace?

Reply:
I'd prefer not to have `Web` as its own namespace if possible - if you're not targeting HTML you wouldn't want to reference `.Web`. We should think about this more and make sure it's not going to break the universe, but I was hoping to leave things in the namespaces they are in.